### PR TITLE
Trad/administration

### PR DESCRIPTION
**Pull request by fdechelle**
Translate administration module:

- many structure changes w.r.t. sphinx compliance
- used :orphan: tags to get rid of warning, not fully satisfying solution

### CODE REVIEW

File: source/modules/administration/entities.rst [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Equipments
+~~~~~~~~~~
+
+This tab allows to configure the different dates present in administrative and financial information. The possible automatic actions are:
+
+* filling when item gets a particular status;
+* filling by copying another date;
+* no automatic filling
+
+The option *Software creation entity* allows to redirect softwares creation to another entity at a higher level in the hierarchy. This functionality applies on *all* softwares of the entity; if redirection must be defined only for some softwares, the :doc:`softwares dictionary</modules/administration/dictionnaries>` must be used.
```
Comment (cconard96) [inline source/modules/administration/entities.rst:132]:
```suggestion
The option *Software creation entity* allows to redirect software creation to another entity at a higher level in the hierarchy. This functionality applies on *all* software of the entity; if redirection must be defined only for some software, the :doc:`software dictionary</modules/administration/dictionnaries>` must be used.
```

File: source/modules/administration/rules/rulesmanagement.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  * assigning an equipment to an entity
+  * granting permissions to a user
+  * assigning category to a software
+  * routing tickets to entities
+  * automatic actions when opening ticket
+
+* :doc:`data dictionaries </modules/administration/dictionnaries>` :
+
+  * manufacturers
+  * softwares
```
Comment (cconard96) [inline source/modules/administration/rules/rulesmanagement.rst:29]:
```suggestion
  * software
```

File: source/modules/administration/rules/rulesmanagement.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+GLPI implements a mechanism for routing tickets opened by email, in order to create them in the right entity.
+
+See :doc:`Rules for assigning a ticket opened via a mail collector </modules/administration/rules/mailcollector>`
+
+Rules for assigning authorizations to a user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See :doc:`Rules for assigning authorizations to a user </modules/administration/rules/userauthorizations>`
+
+Rules for assigning a category to softwares
```
Comment (cconard96) [inline source/modules/administration/rules/rulesmanagement.rst:60]:
```suggestion
Rules for assigning a category to software
```

File: source/modules/administration/dictionnaries.rst [hunk 4]
```diff
@@ -0,0 +1,146 @@
+Dictionaries
+=============
+
+.. |import-rule| image:: images/importrule.png
+.. |playrule| image:: images/playrule.png
+
+Dictionary's allow to modify data already existing in GLPI or new data in order to group redundant data.
+
+Dictionaries are based on GLPI :doc:`rules engine </modules/administration/rules/rulesmanagement>` and are available for some types of items (softwares, suppliers, drop-downs). The rules associated with a dictionary will modify values that are either manually inserted or automatically inserted via an inventory tool or via plugins (for instance CSV file injector).
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:9]:
```suggestion
Dictionaries are based on GLPI :doc:`rules engine </modules/administration/rules/rulesmanagement>` and are available for some types of items (software, suppliers, drop-downs). The rules associated with a dictionary will modify values that are either manually inserted or automatically inserted via an inventory tool or via plugins (for instance CSV file injector).
```

File: source/modules/administration/dictionnaries.rst [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+.. warning:: if database is big, the parameter *memory\_limit* in PHP configuration file must be carefully adjusted: processing by a dictionary can be heavy.
+
+.. hint::
+   * it is highly recommended to play rules on a test database and to backup database before production launch of the dictionary or before inserting new rules
+   * a script is available in directory *scripts* of GLPI installation, named *compute\_dictionnary.php*, that allows to launch dictionnary processing in command line mode; this allows to bypass problems of execution limit and provides a significant speedup
+
+Global dictionaries
+~~~~~~~~~~~~~~~~~~~
+
+Softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:38]:
```suggestion
Software
```

File: source/modules/administration/dictionnaries.rst [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
++++++++++
+
+The softwares dictionary modifies software data (name, version, manufacturer) in order to complete or merge software's. It is used to make more coherent equivalent softwares for which name is not identical (for example Mozilla Firefox 3.0 and Mozilla Firefox 3.6) or to add a manufacturer if it is not present.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:41]:
```suggestion
The software dictionary modifies software data (name, version, manufacturer) in order to complete or merge software's. It is used to make more coherent equivalent software for which name is not identical (for example Mozilla Firefox 3.0 and Mozilla Firefox 3.6) or to add a manufacturer if it is not present. 
```

File: source/modules/administration/dictionnaries.rst [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+
+Global dictionaries
+~~~~~~~~~~~~~~~~~~~
+
+Softwares
++++++++++
+
+The softwares dictionary modifies software data (name, version, manufacturer) in order to complete or merge software's. It is used to make more coherent equivalent softwares for which name is not identical (for example Mozilla Firefox 3.0 and Mozilla Firefox 3.6) or to add a manufacturer if it is not present. 
+
+This dictionary allows also to redirect the creation of a software or of a set of softwares in a given entity, by choosing action *Entity* and to select the entity in which to create the software. This functionality can also be used with general option *Entity for software* available in entity configuration.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:43]:
```suggestion
This dictionary allows also to redirect the creation of a software or of a set of software in a given entity, by choosing action *Entity* and to select the entity in which to create the software. This functionality can also be used with general option *Entity for software* available in entity configuration.
```

File: source/modules/administration/dictionnaries.rst [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+.. note:: for an optimal management of softwares and licenses in a multi-entity environment, it is possible to use the softwares dictionnary simultaneously with softwares visible in sub-entities and thus to use grouping functionality.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:45]:
```suggestion
.. note:: for an optimal management of software and licenses in a multi-entity environment, it is possible to use the software dictionary simultaneously with software visible in sub-entities and thus to use grouping functionality.
```

File: source/modules/administration/dictionnaries.rst [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+.. warning:: using action *Add regexp result* on a version must be used with maximum care; indeed, this action is only taken into account when importing data coming from an inventory tool and will be ignored when re-applying dictionary on an existing database.
+
+As an example, following image describes grouping Mozilla softwares. Criteria are accumulated with AND.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:49]:
```suggestion
As an example, following image describes grouping Mozilla software. Criteria are accumulated with AND. 
```

File: source/modules/administration/dictionnaries.rst [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+This dictionary allows also to redirect the creation of a software or of a set of softwares in a given entity, by choosing action *Entity* and to select the entity in which to create the software. This functionality can also be used with general option *Entity for software* available in entity configuration.
+
+.. note:: for an optimal management of softwares and licenses in a multi-entity environment, it is possible to use the softwares dictionnary simultaneously with softwares visible in sub-entities and thus to use grouping functionality.
+
+.. warning:: using action *Add regexp result* on a version must be used with maximum care; indeed, this action is only taken into account when importing data coming from an inventory tool and will be ignored when re-applying dictionary on an existing database.
+
+As an example, following image describes grouping Mozilla softwares. Criteria are accumulated with AND. 
+
+.. figure:: images/critereMozilla.png
+   :alt: Criteria for grouping Mozilla softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:52]:
```suggestion
   :alt: Criteria for grouping Mozilla software
```

File: source/modules/administration/dictionnaries.rst [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+   :align: center
+
+   Criteria for grouping Mozilla softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:55]:
```suggestion
   Criteria for grouping Mozilla software
```

File: source/modules/administration/dictionnaries.rst [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+
+The following image describes associated action for grouping Mozilla softwares.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:57]:
```suggestion
The following image describes associated action for grouping Mozilla software.
```

Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:57]:
```suggestion
The following image describes associated action for grouping Mozilla software.
```

File: source/modules/administration/dictionnaries.rst [hunk 13]
```diff
@@ -0,0 +1,146 @@
...
+
+.. figure:: images/actionMozilla.png
+   :alt: Action for grouping Mozilla softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:60]:
```suggestion
   :alt: Action for grouping Mozilla software
```

File: source/modules/administration/dictionnaries.rst [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+   :align: center
+
+   Action for grouping Mozilla softwares
+
+The following image presents result of grouping Mozilla software: softwares are grouped by type (Mozilla Thunderbird, Mozilla Firefox...) and versions are grouped by type.
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:65]:
```suggestion
   Action for grouping Mozilla software

The following image presents result of grouping Mozilla software: software are grouped by type (Mozilla Thunderbird, Mozilla Firefox...) and versions are grouped by type.
```

File: source/modules/administration/dictionnaries.rst [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+
+.. figure:: images/resultatMozilla.png
+   :alt: Result of grouping Mozilla softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:68]:
```suggestion
   :alt: Result of grouping Mozilla software
```

File: source/modules/administration/dictionnaries.rst [hunk 16]
```diff
@@ -0,0 +1,146 @@
...
+   :align: center
+
+   Result of grouping Mozilla softwares
```
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:71]:
```suggestion
   Result of grouping Mozilla software
```

File: source/modules/administration/dictionnaries.rst (hunk 4 above)
Comment (cconard96) [inline source/modules/administration/dictionnaries.rst:9]:
```suggestion
Dictionaries are based on GLPI :doc:`rules engine </modules/administration/rules/rulesmanagement>` and are available for some types of items (software, suppliers, drop-downs). The rules associated with a dictionary will modify values that are either manually inserted or automatically inserted via an inventory tool or via plugins (for instance CSV file injector).
```

**Review by cconard96 — CHANGES_REQUESTED** [review]
Two changes here:

1. Change equipments to assets, or alternatively just equipment (Equipment is singular and plural). Having "Assets" seems to match the other translations though.
2. Change softwares to software (Software is singular and plural)

**Review by cconard96 — APPROVED** [review]

File: source/modules/administration/maintenance.rst [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+------------------------------
+
+It is possible to backup the database in SQL or XML directly in GLPI and restore this backup later.
+
+.. hint:: It is recommended to use SQL format.
+
+The list of backups appears when at least one backup has been performed. It is then possible to delete a previously made backup, to restore the database from the backup or to download the backup to save ut externaly.
+
+.. warning:: the restore overwrites all changes made between the backup date and the restore date.
+
```
Comment (trasher) [inline source/modules/administration/maintenance.rst:26]:
```suggestion
```

This has been dropped from GLPI

File: source/modules/administration/maintenance.rst [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+   GLPI maintenance
+
+.. note:: the different buttons appear according to the authorization granted in profile.
+
+.. hint:: It is highly recommended to use an external tool, such as ``mysqldump``, to make scheduled external backups and to store these backups on external media.
+
+To perform a complete backup of GLPI, it is essential to back up all GLPI files. GLPI files are located in the installation directory on the web server and contain the database backup files created from the `Administration> Maintenance` menu, as well as all the documents added in GLPI (associated with tickets, asset, contracts, etc.).
+
+.. hint:: If GLPI is installed in a ``glpi`` directory, then this directory is the directory to be completely backed up.
```
Comment (trasher) [inline source/modules/administration/maintenance.rst:39]:
```suggestion
```

Embed backup has been dropped; while backup instructions are still relevant; I guess it should not be present here

**Review by trasher — CHANGES_REQUESTED** [review]

File: source/modules/administration/maintenance.rst (hunk 18 above)
Comment (fdechelle) [reply source/modules/administration/maintenance.rst:28]: Yes backup instructions should be somewhere else, not here.

**Review by trasher — APPROVED** [review]

**Pull request merged**